extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/FinalDestinationBuildItem.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.smallrye.graphql.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class FinalDestinationBuildItem extends SimpleBuildItem {
```

Comment:
Maybe make the name more specific to the extension?

Reply:
OK will do